sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtDataType.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Data types in the ADT query language.
```

Comment:
Do you have any web links you can point to?

Reply:
what do you think of [this ](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models#schema)one?

Reply:
looks good